bcda/web/api.go

```diff
@@ -240,9 +274,6 @@ func bulkRequest(resourceTypes []string, w http.ResponseWriter, r *http.Request,
 			return
 		}
 	}
-
-	w.Header().Set("Content-Location", fmt.Sprintf("%s://%s/api/v1/jobs/%d", scheme, r.Host, newJob.ID))
```

Comment:
Needed to pull this into the defer call. If we encounter an issue on the commit, we can notify the caller of this failure.